content/tutorials/getting-started/index.md

```diff
@@ -166,60 +175,113 @@ Now, you can directly skip to the [generate a package]({{< ref "#generate-packag
 
 ### Using the bundle {#create-simulation-bundle}
 
-#### Step 3.1 - Generate a Package {#generate-package}
+#### Step 3.1 - Create an API token
+
+On Gatling Enterprise, click on the [API Tokens section]({{< ref "../../reference/admin/api_tokens/" >}}) and on create. Give a name to your API token, and give him `Configure` as Organization role, then hit save.
 
-Gatling Enterprise requires a package built from a Gatling project in order to run a test.
+Next, copy the content of the API token, you won't be able to access it once you close the modal.
 
-To create a package, run the `enterprisePackage.bat` (if Windows) or `enterprisePackage.sh` (if Linux/MacOS) file, Gatling will start creating your first package:
+#### Step 3.2 - Create and run your first simulation on the cloud
+
+We will use the CLI tool interact with `cloud.gatling.io`, so run it with the API token you just generated : 
```

Comment:
**We will use the CLI tool interact with `cloud.gatling.io**
Why do you mean by that?